_data/events.yml

```diff
@@ -0,0 +1,11 @@
+- name: Contentathon Day 1
```

Comment:
Was the idea of using json formatted according to schema.org explored? The content can then be validated using  https://search.google.com/test/rich-results

Reply:
I putted recently adding schema.org for the contributors on the agenda, but this could indeed also be used for the events thanks!

Reply:
If there is an issue, happy to have a go at it if it helps

Reply:
@bedroesb I had a go last night at using schema.org for contributors (see https://github.com/jburel/rdmkit/tree/contributors evaluation at that stage) I was curious about the logic in place to handle contributors in https://github.com/elixir-europe/rdmkit/blob/master/_includes/contributor-page.html and other contributor pages.
The approach taken will no longer work when using schema.org.

Is there a reason not to include all contributors in a json file, this could be a second json if required. This will also avoid propagating the logic to the multiple contributor pages

Reply:
Hello @jburel, I know it is not the nicest logic, but this Is the way that I found to collect all the contributors from the pages (which do not all exist in the yaml file) + al the contributors that exist in the yaml file (which are not all mentioned in the pages). The yaml is only used to extra info or to add contributors that do not contribute to a specific page. This is proven to be the most easy for the end user.

Reply:
Out of experience people do not like to fill in the yaml ....

Reply:
@jburel lets take this discussion to #597